2-configs/ini/task_2_ini.py

```diff
@@ -1,8 +1,29 @@
 import configparser
+import os
 
-config = configparser.ConfigParser()
-config.read('config.ini')
 
-password_value = config.get("keys", "password")
+path_to_ini = 'config.ini'
+search_key = 'password'
+
+
+def exist_ini(exists_ini):
+    if os.path.exists(exists_ini):
+        search_pass(exists_ini, search_key)
+    else:
+        return print(f'file .ini doesnt not exists')
```

Comment:
I would suggest the following change, for more accurate information
```suggestion
        return print(f'{path_to_ini} doesnt not exists')
```